cmd/ak/cmd/runtimes/build.go

```diff
@@ -20,48 +23,65 @@ const (
 var (
 	output   string
 	dir      string
-	paths    []string
 	values   []string
 	describe bool
 )
 
 var buildCmd = common.StandardCommand(&cobra.Command{
-	Use:     "build [--dir=...] <--path=...> [--output=...] [--describe]",
+	Use:     "build [paths] [--dir=...] [--output=...] [--describe]",
```

Comment:
More accurate, but up to you:

```suggestion
	Use:     "build <path> [<path> [...]] [--dir=...] [--output=...] [--describe]",
```

Reply:
fixed